src/backend/project.js

```diff
@@ -0,0 +1,14 @@
+//@flow
+
+import {Graph} from "../core/graph";
+import {TimelineCred} from "../analysis/timeline/timelineCred";
+import {type Project} from "../core/project";
+
+export type ProjectStorageExtras = {
```

Comment:
Why is this file called `project.js` and not `projectStorage.js`?

Reply:
Had "project backend" vs "core project" in mind. You're right though, not having anything else in it it would make more sense as `projectStorage.js`.

Reply:
Renamed